### ISSUE TITLE
Latex parentheses and precedences issues

### ISSUE THREAD

**Issue by ilBarbogio** [davidedc/Algebrite#58]
Hello, don't know if this is an issue with the latex expressions generated, butI havn't found a workaround (I'm using the latest bundle-for-browser js):

let's say I have:     2/x
and I want to obtain this at the power of 3

so in the end I want latex for (2/x)^3 
(I'm making it a two step operation beacause I need to substitute first, it's not really necessary for the issue a hand, you can replicate it with a single string)

I run this in my script:
Algebrite.run("printlatex("+A+")")

but I end up with an unnecessary parenthesis:
![image](https://user-images.githubusercontent.com/26724661/31338700-95597622-ad00-11e7-9893-8a69e5c7b407.png)

It's possible to get rid of the extra parentheses? It isn't much, and it doesn't affect the algebra, but it's a bit annoying to see and aesthetic is paramoount in my project.

But there's more: I'm not the first to have problems with the fact that blocking simplification when parsing to latex isn't so easy (I've found some posts, but only a few... maybe I'm not the first but surely one of the few). With the former example, what if I'd like to obtain the fraction inside \left( and \right) (latex-y speaking)? Hoping the image upload will work, something like this:
![image](https://user-images.githubusercontent.com/26724661/31338989-ae3c4e7a-ad01-11e7-9cf6-f07a9ba84c38.png)

Any help much apreciated, and thanks for the great library! Calcolus and other features are really great for my project

**Comment by davidedc** [comment]
yes I see what you mean for the "unnecessary parenthesis", I'll see what can be done...

As for the second point, you are asking for a particular form that is different from the internal representation of Algebrite... that might be harder, I can't tell if there is a way to systematically "drive" the system to "your" preferred form. You seem to want the x to have a low exponent, or to "collect" the exponent as much as possible... I'd have to see how to "get" to that but it's unclear to me ATM
